### PR TITLE
Use Clojure resolve to resolve symbols in clj

### LINKED ISSUE #90: Linting breaks when using subscribe named function call in defui
**Describe the bug**
When using `subscribe` as a function name in `defui` component, I get a `NullPointerException` during macro expansion. This only seems to happen on the JVM.

If I remove the re-frame linting in `uix.linter`, then macro expansion works again.

**Include minimal repro**
This fails for me on UIx 0.8.1, when compiling via nrepl on the JVM.

```clojure
(defn subscribe [ref]
  ref)

(defonce *state (atom 1))

(defui foo [a]
  (let [v (subscribe *state)]
    ($ :p a v)))
```

**Expected behavior**
We should be able to use the `subscribe` function name in `defui` on the JVM.

**Additional context**
We don't use re-frame anywhere, but this is own our subscribe implementation.

### PR DESCRIPTION
**Pull request by chrisetheridge**
This fixes #90

### CODE REVIEW

File: core/src/uix/linter.clj [hunk 1]
```diff
@@ -260,10 +260,13 @@
   (read-re-frame-config))
 
 (defn lint-re-frame! [form env]
-  (let [sources (->> (uix.lib/find-form rf-subscribe-call? form)
-                     (keep #(let [v (ana/resolve-var env (first %))]
-                              (when (contains? re-frame-config (:name v))
-                                (assoc v :source %)))))]
+  (let [resolve-fn (if (uix.lib/cljs-env? env)
+                     ana/resolve-var
+                     resolve)
```
Comment (roman01la) [inline core/src/uix/linter.clj:265]: would be good to add a basic test for the linter running on JVM, wdyt?

Comment (chrisetheridge) [reply core/src/uix/linter.clj:265]: Yeah nice idea. Is it fine if the failing cases just crashes the tests? There's no `is-not-thrown?` in Clojure test, and it felt quite heavy adding it just for this case

Comment (roman01la) [reply core/src/uix/linter.clj:265]: Since all linters are throwing, it would be actually great to have such utility! We'll use for other tests as well

Comment (chrisetheridge) [reply core/src/uix/linter.clj:265]: Okay cool, I will add it!

Comment (chrisetheridge) [reply core/src/uix/linter.clj:265]: Tests are here https://github.com/pitch-io/uix/pull/92/commits/a3da57841bd3311207bb007821050a7c26db598c

**Comment by roman01la** [comment]
Thanks!

**Pull request merged**